edit-post/components/header/plugin-sidebar-more-menu-item/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * WordPress dependencies
+ */
+import { compose, createHigherOrderComponent } from '@wordpress/element';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { MenuItem } from '@wordpress/components';
+import { PluginContext } from '@wordpress/plugins';
+
+/**
+ * Internal dependencies
+ */
+import PluginsMoreMenuGroup from '../plugins-more-menu-group';
+
+const PluginSidebarMoreMenuItem = ( { children, isSelected, icon, onClick } ) => (
+	<PluginsMoreMenuGroup>
+		{ ( fillProps ) => (
+			<MenuItem
+				icon={ isSelected ? 'yes' : icon }
+				isSelected={ isSelected }
+				onClick={ compose( onClick, fillProps.onClose ) }
+			>
+				{ children }
+			</MenuItem>
+		) }
+	</PluginsMoreMenuGroup>
+);
+
+export default compose(
+	createHigherOrderComponent(
```

Comment:
Should we offer this as a proper export of `@wordpress/plugins` : `withPluginContext` ?

Reply:
It might make sense to do it, because we use a very similar logic in `PluginSidebar`.